src/steamship/data/plugin.py

```diff
@@ -170,7 +171,7 @@ def create(
         handle: str = None,
         training_platform: Optional[HostingType] = None,
         metadata: Union[str, Dict, List] = None,
-        upsert: bool = None,
+        upsert: bool = True,
```

Comment:
I think I'd prefer default to be False rather than True - not sure we would want to assume clobbering by default on a method called 'create'